api/common/secretbackends/client.go

```diff
@@ -76,7 +77,7 @@ func (c *Client) GetBackendConfigForDrain(backendID *string) (*provider.ModelBac
 		arg.BackendIDs = []string{*backendID}
 	}
 	err := c.facade.FacadeCall(context.TODO(), "GetSecretBackendConfigs", arg, &result)
-	if err != nil && !errors.Is(err, errors.NotFound) {
+	if err != nil && !errors.Is(err, secretbackenderrors.NotFound) {
```

Comment:
And here.

Reply:
For these errors I don't think we need to but doesn't hurt to be safe